feat: explicitly log authentication errors as warnings

luca-gr4vy opened the pull request:
**Description:** Adds `gr4vyId` to the options object passed to the embed, for debugging purposes.

**Ticket:** https://gr4vy.atlassian.net/browse/TA-1026

**Screenshot(s):**

![Screenshot 2022-03-21 at 17 28 41](https://user-images.githubusercontent.com/101414321/159519684-dbc7e994-2c3f-419b-b782-f5a61850d89f.png)

douglaseggleton reviewed (approved):
Looks good to me :-) Before you hit merge though - See https://github.com/gr4vy/gr4vy-embed/blob/main/CONTRIBUTING.md#releasing-a-pr (I think `internal` is the correct label)

luca-gr4vy commented:
> @luca-gr4vy I think this works, but doesn't EmbedUI know the ID from the host it's loaded on? Not sure what makes more sense.

@cbetta the following is taken from the dashboard codebase (it's how the Integrations page shows the id)

```
const getGr4vyId = (): string => {
  if (isDev()) {
    return 'dev'
  } else if (isSandbox()) {
    return document.location.host.split('.')[1]
  }
  return document.location.host.split('.')?.[0]
}
```

though, I believe getting it directly from the embed's config should be more robust (?)

cbetta commented:
> > @luca-gr4vy I think this works, but doesn't EmbedUI know the ID from the host it's loaded on? Not sure what makes more sense.
> 
> @cbetta the following is taken from the dashboard codebase (it's how the Integrations page shows the id)
> 
> ```
> const getGr4vyId = (): string => {
>   if (isDev()) {
>     return 'dev'
>   } else if (isSandbox()) {
>     return document.location.host.split('.')[1]
>   }
>   return document.location.host.split('.')?.[0]
> }
> ```
> 
> though, I believe getting it directly from the embed's config should be more robust (?)

Agreed. Keep it as-is.
